src/Build/BackEnd/Components/SdkResolution/SdkResolution.cs

```diff
@@ -63,7 +63,12 @@ internal SdkResolution(SdkResolverLoader sdkResolverLoader)
                     try
                     {
                         var result = (SdkResultImpl)sdkResolver.Resolve(sdk, context, resultFactory);
-                        if (result != null && result.Success)
+                        if (result == null)
+                        {
+                            continue;
```

Comment:
Can you update the docs for the resolver execute method to specify that returning null means the intent to skip?